app/src/components/report/ReportEdit.tsx

```diff
@@ -0,0 +1,209 @@
+                <Form.Control
+                    type="text"
+                    placeholder="Please enter your address"
+                    defaultValue={report?.address}
+                    onChange={(event) => updateReport({ address: event.target.value })}
+                />
+            </Form.Group>
+
+            <Form.Group className="mb-3" controlId="EditReport.GpsCoordinates">
+                <Form.Label>GPS Coordinates</Form.Label>
+                <Form.Control
+                    type="text"
+                    placeholder="Please enter your GPS Coordinates"
+                    defaultValue={report?.gpsCoordinates}
+                    onChange={(event) => updateReport({ gpsCoordinates: event.target.value })}
+                />
+            </Form.Group>
+
+            <Form.Group className="mb-3" controlId="EditReport.IssueDescription">
+                <Form.Label>Issue Description</Form.Label>
+                <Form.Control
+                    as="textarea"
+                    rows={3}
+                    placeholder="Please enter your issue description"
+                    defaultValue={report?.issueDescription}
+                    onChange={(event) => updateReport({ issueDescription: event.target.value })}
+                />
+            </Form.Group>
+
+            <Form.Group className="mb-3" controlId="EditReport.Attachments">
+                <Form.Label>Attachments</Form.Label>
+                <Form.Control
+                    type="text"
+                    placeholder="Please enter your attachments"
+                    defaultValue={report?.attachments}
+                    onChange={(event) => updateReport({ attachments: event.target.value })}
+                />
+            </Form.Group>
+
+            <Form.Label>Status Updates</Form.Label>
+
+            <Form.Group className="mb-3" controlId="EditReport.Email">
+                <Form.Check
+                    type="checkbox"
+                    label="Email"
+                    checked={report?.email}
+                    onChange={(event) => {
+                        updateReport({
+                            email: event.target.checked,
+                        });
+                    }}
+                />
+            </Form.Group>
+
+            <Form.Group className="mb-3" controlId="EditReport.Sms">
+                <Form.Check
+                    type="checkbox"
+                    label="SMS"
+                    checked={report?.sms}
+                    onChange={(event) => {
+                        updateReport({
+                            sms: event.target.checked,
+                        });
+                    }}
+                />
+            </Form.Group>
+
+            <Form.Group className="mb-3" controlId="EditReport.StatusOfReport">
+                <Form.Label>Status Of Report</Form.Label>
+                {['Submitted', 'In Progress', 'Completed', 'On Hold', 'Rejected'].map(
```

Comment:
Can you please create an enum with these categories? So we can easily use the sam evalues on the frontend and backend :)